store/store.go

```diff
@@ -444,6 +445,10 @@ func (s *Store) hashToPath(datahash share.DataHash, ext string) string {
 	return filepath.Join(s.basepath, blocksPath, datahash.String()) + ext
 }
 
+func (s *Store) hashToRelativePath(datahash share.DataHash, ext string) string {
```

Comment:
Nit: doesnt need to be a method

Reply:
Made it as a method to be consistent with `hashToPath` and `heightToPath`